igvc_utils/include/igvc_utils/NodeUtils.hpp

```diff
@@ -29,11 +30,37 @@ void getParam(const ros::NodeHandle &pNh, const std::string &param_name, T &para
 /**
 Calculates euclidian distance between two points
 */
-inline double get_distance(double x1, double y1, double x2, double y2)
+template<typename T>
+inline T get_distance(T x1, T y1, T x2, T y2)
 {
   return sqrt(pow(x2 - x1, 2) + pow(y2 - y1, 2));
```

Comment:
Consider replacing with [std::hypot](https://en.cppreference.com/w/cpp/numeric/math/hypot)

Reply:
A classic case of reinventing the wheel. I'll implement this